receiver/sapnetweaverreceiver/metadata.yaml

```
@@ -30,102 +37,174 @@ metrics:
     attributes: []
     enabled: true
-  sapnetweaver.host.memory.virtual.overhead:
-    description: Virtualization System Memory Overhead.
-    unit: "bytes"
+  sapnetweaver.system.availability:
+    description: The system availability percentage.
+    unit: "%"
     gauge:
       value_type: int
+      input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.host.memory.virtual.swap:
-    description: Virtualization System Swap Memory.
-    unit: "bytes"
+  sapnetweaver.system.utilization:
+    description: The system utilization percentage.
+    unit: "%"
     gauge:
       value_type: int
+      input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.http.count:
-    description: The number of HTTP Sessions.
-    unit: "{sessions}"
+  sapnetweaver.memory.usage:
+    description: The memory usage percentage.
+    unit: "%"
+    gauge:
+      value_type: int
+      input_type: string
+    attributes: []
+    enabled: true
+  sapnetweaver.memory.configured:
+    description: The amount of configured memory.
+    unit: "By"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+    enabled: true
+  sapnetweaver.memory.free:
+    description: The amount of free memory.
+    unit: "By"
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
-      input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.security.count:
-    description: The number of Security Sessions.
-    unit: "{sessions}"
+  sapnetweaver.session.count:
+    description: The amount of of sessions created.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.web.count:
-    description: The number of Web Sessions.
-    unit: "{sessions}"
+  sapnetweaver.queue.count:
+    description: The queue length.
+    unit: "{entr}"
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.browser.count:
-    description: The number of Browser Sessions.
-    unit: "{sessions}"
+  sapnetweaver.queue_peak.count:
+    unit: "{entr}"
+    description: The peak queue length.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.sessions.ejb.count:
-    description: The number of EJB Sessions.
-    unit: "{sessions}"
+  sapnetweaver.job.aborted:
+    description: The amount of aborted jobs.
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
       input_type: string
     attributes: []
     enabled: true
-  sapnetweaver.icm_availability:
-    description: ICM Availability (color value from alert tree).
+  sapnetweaver.abap.update.errors:
```

This should be 
```suggestion
  sapnetweaver.abap.update.error.count:
```

No plurals in naming with a unit of `{errors}`

---

updated